receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -217,7 +236,6 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 		// Instead of creating a whole new OTLP metric, we just append the new sample to the existing OTLP metric.
 		// This cache is called "intra" because in the future we'll have a "interRequestCache" to cache resourceAttributes
 		// between requests based on the metric "target_info".
```

Comment:
Could we also delete those coments above the variable?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/bc70d0c56021018f6ce87fe7ecc2beda13e886f3